tests/spread/smoketests/metafiles/charmcraft_yaml/task.yaml

```
@@ -0,0 +1,102 @@
+summary: create a charm that use charmcraft.yaml only
+
+include: 
+  - tests/
+
+prepare: |
+  tests.pkgs install unzip
+  charmcraft init --project-dir=charm
+  rm -f charm/metadata.yaml
+  rm -f charm/config.yaml
+
+  cat <<- EOF > charm/charmcraft.yaml
+  name: test-charm-name
+  type: charm
+  summary: test-summary
+  description: test-description
+
+  bases:
+    - build-on:
+      - name: ubuntu
+        channel: "22.04"
+    run-on:
+      - name: ubuntu
+        channel: "22.04"
+
+  title: test-title
+
+  links:
+    issues: https://example.com/issues
+    contact:
+      - https://example.com/contact
+      - contact@example.com
+      - "IRC #example"
+    documentation: https://example.com/docs
+    source:
+      - https://example.com/source
+      - https://example.com/source2
+      - https://example.com/source3
+    website:
+      - https://example.com/
+
+  actions:
+    pause:
+      description: Pause the database.
+    resume:
+      description: Resume a paused database.
+    snapshot:
+      description: Take a snapshot of the database.
+      params:
+        filename:
+          type: string
+          description: The name of the snapshot file.
+        compression:
+          type: object
+          description: The type of compression to use.
+          properties:
+            kind:
+              type: string
+              enum: [gzip, bzip2, xz]
+            quality:
+              description: Compression quality
+              type: integer
+              minimum: 0
+              maximum: 9
+      required: [filename]
+      additionalProperties: false
+
+  options:
+    test-int:
+      default: 123
+      description: test-1
+      type: int
+    test-string:
+      description: test-2
+      type: string
+    test-float:
+      default: 1.23
+      type: float
+    test-bool:
+      default: true
+      type: boolean
+
+  EOF
+
+restore: |
+  pushd charm
+  charmcraft clean
+  popd
+  
+  rm -rf charm
+
+execute: |
+  cd charm
+  charmcraft pack --verbose
+  test -f charm*.charm
+  unzip -l charm*.charm | grep "actions.yaml"
+  unzip -l charm*.charm | grep "config.yaml"
+  unzip -l charm*.charm | grep "metadata.yaml"
```

```suggestion
  unzip -l charm*.charm | MATCH "actions.yaml"
  unzip -l charm*.charm | MATCH "config.yaml"
  unzip -l charm*.charm | MATCH "metadata.yaml"
```

Uses spread's [built-in MATCH function](https://github.com/snapcore/spread#functions)